lib/domain/entity/component/data_object_component.dart

```diff
@@ -99,24 +99,8 @@ class DataObjectComponent with _$DataObjectComponent {
         fileReference.getTypeDeclaration(DataFileType.response);
     final requestName = fileReference.getTypeDeclaration(DataFileType.request);
     final entityName = fileReference.getTypeDeclaration(DataFileType.entity);
-
     final codeLines = List<String>.empty(growable: true);
-
-    ///Add common imports
-    if (createResponseToEntityMapper) {
-      codeLines.add(
-        "import 'package:$projectName/${fileReference.getFileImportName(DataFileType.response)}';",
-      );
-    }
-    if (createEntityToRequestMapper) {
-      codeLines.add(
-        "import 'package:$projectName/${fileReference.getFileImportName(DataFileType.request)}';",
-      );
-    }
-    codeLines
-      ..add(
-        "import 'package:$projectName/${fileReference.getFileImportName(DataFileType.entity)}';",
-      )
+    final imports = <String>{}
```

Comment:
@cozvtieg9, I added additional logic to clean up duplicates a bit, since this doesn't completely clean up duplicates or unused imports, then I can roll the commit back and we'll revisit this in future releases